model/jpa/src/main/java/org/keycloak/models/jpa/session/JpaUserSessionPersisterProviderFactory.java

```diff
@@ -43,6 +44,11 @@ public void init(Config.Scope config) {
 
     }
 
+    @Override
```

Comment:
I suggest also removing the `default` method from the `UserSessionPersisterProviderFactory`.

Reply:
The `UserSessionPersisterProviderFactory` is empty. Which `default` do you mean?

Reply:
Ah, my bad, it's already done. Please ignore this comment :-)